src/ecs/rendering/resources.rs

```diff
@@ -135,3 +137,33 @@ impl AsRef<Rgba> for AmbientColor {
         &self.0
     }
 }
+
+/// This specs resource with id 0 permits sending commands to the
+/// renderer internal window.
+pub struct WindowCommander {
+    // It's unlikely we'll get more than two commands per frame
+    // 2 Boxes also make this the same size as a Vec, so this costs
+    // no more space in the structure than a Vec would.
+    //
+    // NOTE TO FUTURE AUTHORS: This could be an FnOnce but that's not possible
+    // right now as of 2017-10-02 because FnOnce isn't object safe.  It might
+    // be possible as soon as FnBox stabilizes.  For now I'll use FnMut instead.
+    pub(crate) queue: SmallVec<[Box<FnMut(&Window) + Send + Sync + 'static>; 2]>,
+}
+
+impl WindowCommander {
```

Comment:
Meh, this name sounds more like a strategy RTS game's name. Could we rename it? Maybe `WindowMessages`?

Reply:
Hahaha 😆 "Coming soon to stores near you!  Window Commander!!  Tell panes of glass to do things which they'll proceed to ignore because they're just glass!  Copyright 2017"

I'll rename it :P